actions/llm-action-error-reporter/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "llm-action-error-reporter",
+  "version": "0.1.0",
```

Comment:
you can leave this at `0.0.0` and the changeset will bump it to `0.1.0` when it's released :)

Reply:
thank you @momentmaker ! updated.